tests/unit/lib/charms/kubernetes_charm_libraries/v0/test_multus.py

```diff
@@ -1169,6 +1169,16 @@ def test_given_nads_not_created_when_nad_config_changed_then_patch_statefulset_i
             privileged=False,
         )
 
+    @patch("lightkube.core.client.GenericSyncClient", new=Mock)
+    @patch(f"{MULTUS_LIBRARY_PATH}.KubernetesClient.unpatch_statefulset")
+    def test_statefulset_unpatched_on_remove(self, patch_unpatch_statefulset):
```

Comment:
Please follow the test naming conventions used in all the other tests (given x, when y, then z)